File: problem009/9.rb
#!/usr/bin/env ruby
(1..1000).each{|b|
   (1..b).each{|a|
      c = Math.sqrt(a**2 + b**2)
      if a+b+c == 1000.0 and (a < b) and (b < c) then
         puts a*b*c.to_i
         exit
      end
   }
}

File: problem009/9.js
#!/usr/bin/env node

var a, b, c;

for (b = 1; b <= 500; b++) {
    for (a = 1; a <= b; a++) {
        c = Math.sqrt(Math.pow(a, 2) + Math.pow(b, 2));
        if ((a + b + c == 1000) && (a < b) && (b < c)) {
            console.log(a * b * c);
            return;
        }
    } 
}
